crates/bevy_render/src/render_graph/nodes/pass_node.rs

```diff
@@ -235,7 +235,7 @@ where
                     // attempt to draw each visible entity
                     let mut draw_state = DrawState::default();
                     for visible_entity in visible_entities.iter() {
-                        if query_state.get(world, visible_entity.entity).is_err() {
+                        if query_state.get(world, visible_entity.entity, world.get_exclusive_system_counter(), world.get_global_system_counter()).is_err() {
```

Comment:
This api change is a pretty significant downgrade in usability (way nastier to type and look at) and error-protection (callers might pass in the wrong value when they really shouldn't need to worry about it).

Reply:
Fixed in 7d38cc4426abe99143cc479bf5befb63de8020c0